addons/sale_crm/crm_lead.py

```diff
@@ -11,6 +17,30 @@ def _get_sale_amount_total(self, cr, uid, ids, fields, args, context=None):
             res[key] = value
         return res
 
+    def action_view_quotation(self, cr, uid, ids, context=None):
+    # '''
+    # This function returns an action that display existing quotations of given opportunity ids. It can either be a in a list or in a form view, if there is only one quotation to show.
+    # '''
+        mod_obj = self.pool.get('ir.model.data')
```

Comment:
Use self.pool['ir.model.data'] instead of self.pool.get. Indeed if ir.model.data is not in the pool, you want it to crash now, not later with some weird error :) .